core/src/main/java/GameObjects/GameSettings.java

```diff
@@ -0,0 +1,56 @@
+package GameObjects;
+
+import Scenes.Lobby;
+
+/**
+ * The GameSettings class will store the information for the preferences that were selected in the
+ * lobby screen and will have methods that return values corresponding to the values needed in the
+ * buildUrl method of the UrlBuilderImp class.
+ */
+public class GameSettings {
+  private final String amount;
+  private final String category;
+  private final String difficulty;
+
+  public GameSettings(String amount, String category, String difficulty) {
+    this.amount = amount;
+    this.category = category;
+    this.difficulty = difficulty;
+  }
+
+  public String getCategoryValue() {
+    if (category.compareTo(Lobby.GENERAL_TEXT) == 0) {
+      return "9";
+    } else if (category.compareTo(Lobby.GEOGRAPHY_TEXT) == 0) {
+      return "22";
+    } else if (category.compareTo(Lobby.MUSIC_TEXT) == 0) {
+      return "12";
+    } else if (category.compareTo(Lobby.VIDEOGAMES_TEXT) == 0) {
+      return "15";
+    } else if (category.compareTo(Lobby.SCIENCE_NATURE_TEXT) == 0) {
```

Comment:
I think it would be reasonable, since these constants are the textual representations of the settings, and the buttons display this textual representation.  I'll leave it up to you.